integrations/huggingface-transformers/README.md

```diff
@@ -13,423 +13,106 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-    Arguments pertaining to what data we are going to input our model for training and eval.
-    """
-    ####################################################################################
-    # Start SparseML Integration
-    ####################################################################################
-    nm_prune_config: Optional[str] = field(
-        default='recipes/noprune1epoch.yaml', metadata={"help": "The input file name for the Neural Magic pruning config"}
-    )
-    do_onnx_export: bool = field(
-        default=False, metadata={"help": "Export model to onnx"}
-    )
-    onnx_export_path: Optional[str] = field(
-        default='onnx-export', metadata={"help": "The filename and path which will be where onnx model is outputed"}
-    )
-    ####################################################################################
-    # End SparseML Integration
-    ####################################################################################
-```
-Use the code below to load SparseML optimizers:
-```python
-## Neural Magic Integration here. 
-optim = load_optimizer(model, training_args)
-steps_per_epoch = math.ceil(len(train_dataset) / (training_args.per_device_train_batch_size*training_args._n_gpu))
-manager = ScheduledModifierManager.from_yaml(data_args.nm_prune_config)
-optim = ScheduledOptimizer(optim, model, manager, steps_per_epoch=steps_per_epoch, loggers=None)
-```
-Modify the Hugging Face trainer to take the SparseML optimzier as shown below:
-```python
-# Initialize our Trainer and continue to use your regular transformers trainer
-trainer = QuestionAnsweringTrainer(
-    model=model,
-    args=training_args,
-    train_dataset=train_dataset if training_args.do_train else None,
-    eval_dataset=validation_dataset if training_args.do_eval else None,
-    eval_examples=datasets["validation"] if training_args.do_eval else None,
-    tokenizer=tokenizer,
-    data_collator=data_collator,
-    post_process_function=post_processing_function,
-    compute_metrics=compute_metrics,
-    optimizers=(optim, None), # This is what is new.
-)
-```
-Finally, export the model. It is worth noting that you will have to create a sample batch which will be task-dependent. The code shown below is specific for SQuAD-style Question Answering:
-```python
-exporter = ModuleExporter(
-    model, output_dir='onnx-export'
-)
-sample_batch = convert_example_to_features(
-    datasets["validation"][0],
-    tokenizer,
-    data_args.max_seq_length,
-    data_args.doc_stride,
-    data_args.max_query_length,
-)
-exporter.export_onnx(sample_batch=sample_batch)
+For example, the following command can be run from within the integration's folder to export a trained/sparsified model's checkpoint:
+```bash
+python transformers/examples/pytorch/question-answering/run_qa.py \
+  --model_name_or_path MODELS_DIR/bert-base-12layers_prune80 \
+  --dataset_name squad \
+  --do_eval \
+  --per_device_eval_batch_size 64 \
+  --max_seq_length 384 \
```

Comment:
Can you help add this option: --output_dir MODELS_DIR/bert-base-12layers_prune80/eval